src/components/Spinner.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+/**
+ * Loading animation from:
+ * https://loading.io/css/
+ *
+ * And implemented using style-components animations:
+ * https://styled-components.com/docs/basics#animations
+ */
+
+const Animation = keyframes`
+  0% {
+    top: 8px;
+    height: 64px;
+  }
+  50%, 100% {
+    top: 24px;
+    height: 32px;
+  }
+`
+
+const Container = styled.div`
+  display: inline-block;
+  position: relative;
+  width: 80px;
+  height: 80px;
+  & div{
+    display: inline-block;
+    position: absolute;
+    left: 8px;
+    width: 16px;
+    background: var(--brightfucsia);
+    animation: ${Animation} 1.2s cubic-bezier(0, 0.5, 0.5, 1) infinite;
+  }
+  & div:nth-child(1) {
+    left: 8px;
+    animation-delay: -0.24s;
+  }
+  & div:nth-child(2) {
+    left: 32px;
+    animation-delay: -0.12s;
+  }
+  & div:nth-child(3) {
+    left: 56px;
+    animation-delay: 0;
+  }
+`;
+
+const Spinner = () => {
+  return (<Container><div></div><div></div><div></div></Container>)
```

Comment:
Lol, why 3 divs? Just guaranteeing a certain amount of space?

Reply:
just what the spinner required. The link to the actual source is there.